tests/e2e-inst-signatures/e2e-proctree_data_source.go

```diff
@@ -161,6 +162,9 @@ func (sig *e2eProcessTreeDataSource) checkThread(eventObj *trace.Event) error {
 	if threadInfo.Pid != eventObj.HostProcessID {
 		return fmt.Errorf(debug("no match for pid"))
 	}
+	if threadInfo.Timestamp != queryTime {
+		return fmt.Errorf(debug("no match for info timestamp"))
+	}
```

Comment:
Code is correct. Logic is correct, etc. But, why ? The issue does not describe the "why" very well.

If you query for a process, or a thread, or a tree, at a specific point in time, you will get the nearest changelog timestamp item. Why there is a need to know the exact point in time ? (Im not saying I'm against). What would be the benefit ?